README.md

```diff
@@ -1,16 +1,16 @@
-What is deal.II?
+What is deal.III?
```

Comment:
On second thought. I think this is getting a bit unwiedly. What about using `U+00B3` instead?
I think `deal³` looks much nicer.

Reply:
deal-cube it is!